packages/mobile/src/components/add-to-playlist-drawer/AddToPlaylistDrawer.tsx

```diff
@@ -77,6 +80,12 @@ export const AddToPlaylistDrawer = () => {
 
   const userPlaylists = user?.playlists ?? []
 
+  const playlistTrackIdMap = userPlaylists.reduce((acc, playlist) => {
```

Comment:
nit but this will change every time causing the callback to always change. useMemo basically? one other option is to update the selector to just give the id map if that's all we care about?